### DISCUSSION TITLE
Add a buffering circle after hitting play/pause.

### DISCUSSION THREAD

**Discussion by Davis8483** [music-assistant/support#2919] [Feature requests and ideas]
Often times after hitting play/pause there might be a second delay, sometimes even longer if there's an issue. It would be nice to instead show instant feedback to the user that the button was interacted with by replacing it with a buffering circle.

**Comment by OzGav** [comment]
This is available in the 2.9 beta

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
